bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2504,7 +2504,7 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( IsTemplatedSTLClass( name, "vector" ) ) {
+   else if ( IsTemplatedSTLClass( name, "vector" ) || (name.find("RVec<") != std::string::npos) ) {
```

Comment:
Would this match something like `PARVec<float>`?

Reply:
Yes. Unfortunately the whole system in PyROOT for finding pythonizations of specific classes is based on such string magic.

Reply:
Well, we should at least buy a bit of insurance by matching whole things (i.e. matching beginning of word followed by the class name we want). I think it is a bug that this matches more than what it is intended to match.

Reply:
I agree. Let me match on `VecOps::RVec<`, which should be pretty safe.

Reply:
Unfortunatly I cannot match on the full typename because we want to apply it on all templates.